packages/code-du-travail-api/src/server/routes/annuaire/index.js

```diff
@@ -0,0 +1,66 @@
+const Router = require("koa-router");
+const fetch = require("node-fetch").default;
+const API_BASE_URL = require("../api").BASE_URL;
+
+const elasticsearchClient = require("../../conf/elasticsearch.js");
+const getAnnuaireBody = require("./annuaire.elastic");
+
+const router = new Router({ prefix: API_BASE_URL });
+const index = process.env.ELASTICSEARCH_ANNUAIRE_INDEX || "cdtn_annuaire";
+const routeName = "/annuaire";
+
+// todo @lionelb move to constant file
+const defaultMaxSearchDistance = "30km";
+const adresseApiEndPoint = "https://api-adresse.data.gouv.fr/search";
```

Comment:
IMO those const looks like configurations.
We might consider extracting them from the code.